our-brothers-web/src/app/pages/home-page/home-page.component.scss

```diff
@@ -128,34 +128,92 @@
     width: 250px;
     height: 55px;
     font-size: 36px;
-    font-family: 'arbel';
+    font-family: "arbel";
     margin-top: 15px;
   }
 }
 
-
 @media only screen and (max-width: 700px) {
-
-  .video-background{
+  .video-background {
     background-position: -75px -120px;
   }
 
-  .video-show{
-    .video-show-box{
+  .video-show {
+    .video-show-box {
       width: calc(100% - 50px);
       overflow: hidden;
 
-      #vid{
+      #vid {
         margin-top: 0px;
         width: 245%;
         left: 50%;
       }
     }
   }
 
-  .big-heading{
+  .big-heading {
     font-size: 40px;
     margin-top: 15px;
     margin-bottom: 15px;
   }
+
+  .contact-form {
+    padding: 35px 0;
+    .contact-form-fields {
+      padding: 0px;
+      width: calc(100% - 10px);
+
+      .form-field {
+        width: 45%;
+        margin-left: 7px;
+        margin-right: 7px;
+        margin-bottom: 14px;
+      }
+      .message {
+        width: calc(100% - 20px);
+      }
+    }
+    .contact-send-button {
+      width: 200px;
+      height: 45px;
+      font-size: 25px;
+      margin-top: 7px;
+    }
+  }
+  .site-hr {
+    margin-bottom: 20px;
+  }
+  .site-divider {
+    margin-bottom: 20px;
+  }
+  .small-heading {
+    margin-bottom: 20px;
+  }
+  .medium-heading {
+    margin-bottom: 20px;
+  }
+  .see-more {
+    margin-bottom: 20px;
+    justify-content: center;
+    a {
+      margin-left: 0px;
+    }
+  }
+  .site-quote.mettings-quote {
+    margin-bottom: 20px;
+  }
+  .mettings-gallery {
+    margin-bottom: 0px;
+
+    .circle-top-right {
+      right: -5%;
+      left: auto;
+      transform: none;
```

Comment:
remove

Reply:
need it,
This code defines the location of the circles (in the photo gallery area)